code/modules/mob/living/silicon/ai/ai_verbs.dm

```diff
@@ -237,4 +237,63 @@
 
 	var/datum/browser/popup = new(src, "manifest", "<div align='center'>Crew Manifest</div>", 370, 420)
 	popup.set_content(dat)
-	popup.open(FALSE)
\ No newline at end of file
+	popup.open(FALSE)
+
+
+/mob/living/silicon/ai/verb/show_laws()
+	set category = "Silicon"
+	set name = "Show Laws"
+
+	if(incapacitated())
+		return
+
+	to_chat(src, "<span class='notice'><b>Obey these laws:</b></span>")
+	for(var/i in laws)
+		to_chat(src, "<span class='notice'>[i]</span>")
+
+
+/mob/living/silicon/ai/verb/state_laws()
+	set category = "Silicon"
+	set name = "State Laws"
+
+	if(incapacitated())
+		return
+
+	if(input(src, "Are you sure you want to announce your laws[radiomod ? " over the [radiomod] channel" : ""]?", "State Laws", "Yes", "No") != "Yes")
+		return
+
+	say("[radiomod] Current Active Lawset:")
+
+	var/delay = 1 SECONDS
+	for(var/i in laws)
+		addtimer(CALLBACK(src, .say, "[radiomod] [i]"), delay)
```

Comment:
```suggestion
		addtimer(CALLBACK(src, .proc/say, "[radiomod] [i]"), delay)
```

Reply:
code\modules\mob\living\silicon\ai\ai_verbs.dm:269:error: .proc/say: undefined type path
it either won't compile, or won't lint properly.

Reply:
so use whatever the basepath for say() is, /mob/living/.proc/say or whatever it is

Reply:
lint is complaining because its not typesafe and this could potentially runtime